### DISCUSSION TITLE
Allow activating pause menu buttons and the Skip button via gameplay keys like stable

### DISCUSSION THREAD

**Discussion by Wrongwayscout** [ppy/osu#26883] [Ideas] [answered]
I'm very used to using my gameplay keys to click on the skip intro button, as well as click resume, retry, and exit on the pause menu. It would be really nice if this were implemented to match stable.

**Comment by MikkelCornelius** [comment]
aren't there tablet players with mouse buttons disabled? Can they even skip the intro in lazer?

**Reply by Rekunan** [reply]
Space bar still works.

**Comment by Rekunan** [comment, marked as answer]
Duplicate of #3551
